gadgets/fsnotify/gadget.yaml

```diff
@@ -0,0 +1,117 @@
+name: fsnotify
+description: Gadget enriching inotify or fanotify events
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://inspektor-gadget.io/docs
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
```

Comment:
ah, not related to this PR, but should probably update all gadgets to point to the specific gadget docs and source.

Reply:
Done for other gadgets in #3376